packages/@okta/i18n/src/properties/login.properties

```diff
@@ -644,7 +644,7 @@ password.reset.title.specific = Reset your {0} password
 # {0} is a generated list of requirements, based on translations from the below keys.
 password.complexity.requirements = Password requirements: {0}.
 # {0} is a number
-password.complexity.history = Your password cannot be any of your last {0} passwords.
+password.complexity.history = Your password cannot be any of your last {0} password(s).
```

Comment:
I believe this will cause failures in the snapshot tests we have. You can update them using `yarn workspace v3 jest -u`.

Reply:
@glenfannin-okta  [SIW tests](https://bacon-go.aue1e.saasure.net/commits?artifact=okta-signin-widget&branch=CA-OKTA-649640&page=1&pageSize=6&sha=1b2bd4713550c84eb59be21a2deee226e944f9bd&tab=main) were fine, waiting on okta-core tests.

Reply:
Oh I see, Gen3 doesnt display this requirement (decision from UX). Sorry for the confusion, thank you!